src/lib/components/TokenSelect/index.tsx

```diff
@@ -45,10 +46,25 @@ interface TokenSelectDialogProps {
 }
 
 export function TokenSelectDialog({ value, onSelect }: TokenSelectDialogProps) {
+  const { chainId } = useActiveWeb3React()
```

Comment:
I don't think we should be clearing the query on chain changes - it adds complexity that we don't need to handle, and is out of scope of this change.